pmda/test/test_leaflet.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+from numpy.testing import (assert_almost_equal, assert_array_equal,
+                                   assert_array_almost_equal)
+
+import MDAnalysis
+from MDAnalysisTests.datafiles import (PSF, DCD)
```

Comment:
These datafiles do not include a membrane. 

Use the MDAnalysis tests for a single frame https://github.com/MDAnalysis/mdanalysis/blob/edb1643c0a78f7e5447aa3ccc02712e4205ff073/testsuite/MDAnalysisTests/analysis/test_leaflet.py#L30 to compare results and the files https://github.com/MDAnalysis/mdanalysis/blob/edb1643c0a78f7e5447aa3ccc02712e4205ff073/testsuite/MDAnalysisTests/analysis/test_rdf_s.py#L31 contain a trajectory with a membrane protein in a membrane.

Reply:
May I ask which atoms to select from the large trajectory? Thank you!

Reply:
Most likely "name P*" – sorry, can't check right now.

Reply:
@VOD555 what kind of lipids are in these trajectories and how do you select the headgroup phosphorous?

Reply:
There are POPG and POPE lipids in these trajectories, and you can use 
`u.select_atoms('name P and resname POPG')`